src/main/java/com/fnp/Webapp Fundamentals/newcrud/src/main/java/com/fnp/service/Service.java

```diff
@@ -0,0 +1,87 @@
+package com.fnp.service;
+
+import java.io.IOException;
+
+import javax.servlet.ServletInputStream;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fnp.dao.DBOperation;
+import com.fnp.dto.Employee;
+import com.google.gson.JsonObject;
+
+/**
+ * This class is used for handling all the Business Logic of the Controller
+ * classes.
+ * this one is for Search.
+ * @author Saransh vijay
+ * @category Service Layer class.
+ */
+
+public class Service {
+
+	/**
+	 * ObjectMapper class Of the the Jackson package is used for parsing the
+	 * elements of a string type Json object.
+	 */
+	static ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Create factory class for this to have only a single objectmapper at sevice/project level.